docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
+
+A how to guide for accessing and running applications on a kubernetes cluster.
+
+### Basic Setup
+The kubernetes cluster is accessed via the ```kubectl`` command.  Full details of using the ```kubectl``` command are described [elsewhere](docs/cli.md).  Make sure that ```kubectl``` is in your path by running
+
+```sh
+which kubectl
+```
+
+If that command returns a path to the binary, you are ready to go.  If it doesn't, you need to update your PATH to include the appropriate directory from the Kubernetes directory.
+
+### Commands
+#### run
+```sh
+kubectl run <pod-name> --image=nginx [--replicas=N] [--labels=<key>=<value>,...][--export [--external]]
```

Comment:
I'm in favor of robust config stuff, as described in #1695, but I also FREQUENTLY just want to run a simple container on a cluster.  I either use yaml files I have stashed around or else I fall back on `kubecfg run`.  I am not opposed to having a VERY SIMPLE run command built in.

I might expand this to take multiple --image flags or --images - this can stay very simple but also be useful for quick testing and one-off runs.